<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeTransaction</class>
 <widget class="QDialog" name="MakeTransaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:bold;

}</string>
       </property>
       <property name="text">
        <string>AccountNumber</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_accountNumError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel{

font-weight:bold;

}</string>
       </property>
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_amoutError">
       <property name="styleSheet">
        <string notr="true">QLabel{


color:red;

}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_amount">
       <property name="styleSheet">
        <string notr="true">QLineEdit{


height:30px;
outline:none;
border-radius:5px;
border:1px solid #ccc;
padding-left:15px;
font-size:16px;

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
